src/check_executor.rs

```diff
@@ -185,6 +188,11 @@ fn record_result_metrics(result: &CheckResult) {
     )
     .record((*actual_check_time - *scheduled_check_time).num_milliseconds() as f64);
 
+    tracing::info!(
+        "delay is {}",
+        (*actual_check_time - *scheduled_check_time).num_milliseconds() as f64
+    );
+
```

Comment:
It's duration that is incorrect right?

Reply:
oh you're right lol

Reply:
I was using this to check something else I think